rpc/grpc/build.gradle

```diff
@@ -2,6 +2,7 @@ apply plugin: 'kotlin-kapt'
 
 dependencies {
     implementation project(':heroic-component')
+    implementation project(':heroic-core')
```

Comment:
@hexedpackets when I moved  the `EnvironmentMetadata` to heroic-core i had to add this dependency which seems a bit heavy-handed.

Reply:
Hmm yeah. Before this the dependencies on core are dist and test. I think there's some larger refactoring needed, for now leave it whichever way you prefer (in core or in component).